programmers/3일차/자릿수-더하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(N) {
+    const stringofN = String(N);
+    let sum = 0;
+    
+    for (let number of stringofN.split("")) {
```

Comment:
```suggestion
    for (let number of stringofN) {
```
문자열은  배열처럼 다룰 수 있어요 따라서 굳이 배열로 변환해주지 않으셔도  됩니다